uefi/src/data_types/owned_strs.rs

```diff
@@ -15,6 +15,22 @@ pub enum FromStrError {
     InteriorNul,
 }
 
+impl fmt::Display for FromStrError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "UCS-2 Conversion Error: {}",
+            match self {
+                Self::InvalidChar => "Invalid character",
```

Comment:
@nicholasbishop I think we should find a solution to guarantee consistency. Either we use `derive_more::Display` for errors or the display impl should fall back to the `Debug` impl. What do you think?

Reply:
ok